sql/Pg-upgrade-2.8.31-2.8.32.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS bank_account (
+id INT GENERATED BY DEFAULT AS IDENTITY,
+bic varchar(11),
+iban varchar(255),
+account varchar(255),
+bank varchar(255),
+branch varchar(255),
+country varchar(32),
+country_iso varchar(2),
+city varchar(128),
+state varchar(128),
+zip varchar(11),
+address varchar(128)
+);
```

Comment:
I think that all properties that are not strictly limited by length could be simply defined as `text` instead of `varchar(N)`